examples/plotting/mplcairo_plotting.py

```diff
@@ -0,0 +1,91 @@
+"""
+============================
+Blending maps using mplcairo
+============================
+
+This example shows how to blend two maps using mplcairo.
+
+``matplotlib`` by itself provides only alpha-based transparency for
+superimposing one image onto another, which can be restrictive when trying to
+create visually appealing composite images.
+```mplcairo`` <https://github.com/matplotlib/mplcairo>`__ is an enhancement for
+``matplotlib`` that provides support for
+```cairo``'s compositing operators <https://www.cairographics.org/operators/>`__,
+which include a wide range of
+`blend modes <https://en.wikipedia.org/wiki/Blend_modes>`__ for image overlays.
+
+.. note::
+   This example requires ```mplcairo`` <https://github.com/matplotlib/mplcairo>`__
+   to be installed. While installation via ``pip`` will work in most cases, you
+   may need to refer to
+   `OS-specific notes <https://github.com/matplotlib/mplcairo#installation>`__.
+"""
+
+###############################################################################
+# We need to tell ``matplotlib`` to use a backend from ``mplcairo``.  The
+# backend formally needs to be set prior to importing ``matplotlib.pyplot``.
+# The ``mplcairo.qt`` GUI backend should work on Linux and Windows, but
+# you will need to do something different on macOS or Jupyter (see
+# `their documentation <https://github.com/matplotlib/mplcairo#use>`__).
+
+import matplotlib
+
+if matplotlib.get_backend() == "agg":  # noqa
```

Comment:
What happens with these noqa removed?

Reply:
Ha, hilarious, I didn't realize that it worked that way.  Not only are the `if` and `else` lines not considered out of place – which makes sense since they could be used for conditional imports – the `matplotlib.use()` calls are not considered out of place simply because they are now "hidden" in the `if`..`else` statement.  So, I can remove all of the `noqa` comments.